lib/auth/auth_with_roles_test.go

```diff
@@ -60,6 +64,75 @@ func TestSSOUserCanReissueCert(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// TestGenerateDatabaseCert makes sure users and services with appropriate
+// permissions can generate certificates for self-hosted databases.
+func TestGenerateDatabaseCert(t *testing.T) {
+	ctx := context.Background()
```

Comment:
```suggestion
        t.Parallel()
	ctx := context.Background()
```
Can this test be run in parallel?